### PR TITLE
Add support for WorkflowExecutionUpdateRequested event

### PR DESCRIPTION
**Pull request by Alex-Tideman**
### How was this tested 👻 

- [ ] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### CODE REVIEW

File: src/lib/utilities/is-event-type.ts
Comment (laurakwhit) [inline src/lib/utilities/is-event-type.ts:120]: Do we want to add these to `all-event-types.json` and add tests to `get-group-id.test.ts` as well?

Comment (Alex-Tideman) [reply src/lib/utilities/is-event-type.ts:120]: Yup good call. Going to add those just as soon as I can figure out how to actually run a workflow with this new type (it's on a forked branch using a in-progress temporal server version).

**Comment by dandavison** [comment]
Thanks very much for doing this work. I've tried testing this locally and the UI page loading the new event is showing errors, but there are a lot of moving parts and this may have been an error on my part. Sorry for holding this up but I think it's best if we wait until I get some feature branches merged first so that we can test this more sanely.

**Comment by dandavison** [comment]
I've tested this (briefly, so far) against my server feature branch and can confirm that it is correctly displaying `WorkflowExecutionUpdateRequested` in the history list view and that there are no network fetch errors or JS errors in the console.

image

**Review by stevekinney — APPROVED** [review]

**Pull request merged**
